include/seastar/util/log.hh

```diff
@@ -504,8 +504,8 @@ enum class logger_timestamp_style {
 /// \brief Output stream to use for logging.
 enum class logger_ostream_type {
     none,
-    stdout,
-    stderr,
+    cout,
+    cerr,
```

Comment:
might want to drop this commit and rebase onto the master HEAD.

Reply:
yes will drop and rebase